res/TensorFlowLiteRecipes/Quant_Split_Add_000/test.qconf.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Please split `compiler/*` and `res/*` in different PRs. We have a rule not to mix module of `compiler/*`.

Reply:
I will mark this pr as a draft, and divide it into several (`compiler/luci`, `compiler/circle-quantizer-dredd-recipe-test` and `res`)